packages/std/src/pagination.rs

```diff
@@ -0,0 +1,12 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use crate::{Binary, Uint64};
+
+/// Simplified version of the PageRequest type for pagination from the cosmos-sdk
+#[derive(Serialize, Deserialize, Clone, Default, Debug, PartialEq, Eq, JsonSchema)]
+pub struct PageRequest {
+    pub key: Option<Binary>,
+    pub limit: Uint64,
```

Comment:
I suggest making this an u32. Then it's a JSON number instead of a string, which makes it easier to hadle in Go. u64 would work too but many JSON implementations don't support integers > 53 bit.